Makefile

```diff
@@ -104,6 +104,8 @@ PACKER_TARGET?=virtualbox-iso.cos
 #
 ISO?=$(shell ls $(ROOT_DIR)/*.iso 2> /dev/null)
 
+# Used by publish-repo and create-repo for the snapshot-id 
+GIT_COMMIT ?= $(shell git rev-parse HEAD)
```

Comment:
could we also cover tags here? so we automatically get tagged snapshots

Reply:
yes, Im wrapping my head on how to make this work for both normal commits and tagged snapshots...but make it work on the makefile directly instead of relaying on the CI magic...Im not sure how feasible this is :/

Reply:
maybe something around the lines of https://github.com/c3os-io/c3os/blob/master/versions.sh#L3 ?

Reply:
oh I got something in there to change between commit and tag :D